server/runtime/src/test/java/org/infinispan/server/configuration/ServerConfigurationParserTest.java

```diff
@@ -63,9 +76,18 @@ public void testParser() throws IOException {
       assertEquals(11223, socketBindings.get("hotrod").getPort());
       assertEquals(8080, socketBindings.get("rest").getPort());
 
+      // Security realms
+      List<RealmConfiguration> realms = server.security().realms().realms();
+      assertEquals(2, realms.size());
+      RealmConfiguration realmConfiguration = realms.get(0);
+      assertEquals("default", realmConfiguration.name());
+
+      realmConfiguration = realms.get(1);
+      assertEquals("using-credentials", realmConfiguration.name());
```

Comment:
would it be worth to check if the passwords are parsed correctly?
The last assert is failing. The attribute's value is null (probably my mistake since if I changed the password in `Common`, the test is unable to open the keystore...)
```java
assertEquals("strongPassword", realmConfiguration.ldapConfiguration().attributes().attribute("credential").get());
      assertEquals("secret", new String((char[]) realmConfiguration.trustStoreConfiguration().attributes().attribute("keystorePassword").get())); //stores it as char[]
      assertEquals("1fdca4ec-c416-47e0-867a-3d471af7050f", realmConfiguration.tokenConfiguration().oauth2Configuration().attributes().attribute("clientSecret").get());
      assertEquals("password", realmConfiguration.serverIdentitiesConfiguration().sslConfigurations().get(0).keyStore().attributes().attribute("keyPassword").get());
```